app/views/merchant_invoices/show.html.erb

```diff
@@ -3,3 +3,21 @@
 <%= "Created on: #{@invoice.formatted_date}" %>
 <%= "Customer: #{@invoice.customer_full_name}" %>
 
+<p>Items on this Invoice:</p>
+<table id="ItemTable">
```

Comment:
Nice use of the table. I'll have to keep this in mind when I go back to refactor the admin parallel of this US.